internal.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script is used by crc developer or internal CI to build the KAO/KCMO patch images to
+# brew and then mirror built container images to quay.io/crcont. Pull secret should contain
+# the auth for `quay.io/crcont` before providing to this script.
+#    - Since this script uses rhpkg and kinit commands, it is tested on linux (not mac/windows)
+#    - As of now this script works with 4.11 releases because only `rhaos-4.11-rhel-8` branch
+#      created in dist-git and tested.
+# Usage:
+# If you want to build latest candidate stream for 4.11
+#   - ./internal.sh
```

Comment:
For a standalone script, I'd use a name more explicit than `internal.sh`. From the naming, I'd expect it's some internal snc implementation detail, not a standalone script.

Reply:
@cfergeau At the moment I don't have any better name for script, can be changed later once we have something better, shouldn't be blocker.

Reply:
`build-patched-kao-kcmo-images.sh`?
If we don't pick a relatively good name now, we'll forget about it until someone gets confused and asks why it's named this way.

Reply:
Sure I can go with this name